client/overview.jsx

```diff
@@ -1,44 +1,56 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
 import axios from 'axios';
+import Description from './Components/description.jsx';
+import Lodging from './Components/lodging.jsx';
+import Essentials from './Components/essentials.jsx';
+import Amenities from './Components/lodging.jsx';
+import Details from './Components/details.jsx';
 
 class Overview extends React.Component {
   constructor() {
     super();
     this.state = {
-      title: 'Twisselman Ranch'
+      campId: 0,
+      siteName: 'Twisselman Ranch'
     }
+    this.fetchOverview();
   }
 
   fetchOverview() {
-    return axios.get('/overview', { params: { campId: this.state.campId } })
+    return axios.get('/overview/all', { params: { campId: this.state.campId } })
     .then(response => {
       console.log('Response for get overview', response.data);
       const overview = response.data;
-      console.log(overview.name)
       this.setState({
         siteName: overview.name,
+        owner: overview.owner
       });
-      return overview;
+      // console.log(this.state)
     });
   }
 
   componentDidMount() {
-    this.fetchOverview();
+    // this.fetchOverview();
```

Comment:
do you still need this if you're doing your request in your constructor?

Reply:
no, I think I updated this in subsequent code, but good catch!